candle-onnx/src/eval.rs

```diff
@@ -1950,6 +1950,73 @@ fn simple_eval_(
                 let output = input.sign()?;
                 values.insert(node.output[0].clone(), output);
             }
+            // https://onnx.ai/onnx/operators/onnx__OneHot.html
+            "OneHot" => {
+                let indices = get(&node.input[0])?;
+                let orig_shape = get(&node.input[0])?.dims().to_vec();
+                let depth_tensor = get(&node.input[1])?;
+                let values_tensor = get(&node.input[2])?;
+
+                let depth = depth_tensor.to_scalar::<i64>()? as usize;
```

Comment:
Not a huge deal, but according to the [onnx spec](https://onnx.ai/onnx/operators/onnx__OneHot.html), the depth can be a scalar or rank-1 tensor.